code/game/machinery/hologram.dm

```diff
@@ -35,6 +35,7 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 	name = "\improper AI holopad"
 	desc = "It's a floor-mounted device for projecting holographic images. It is activated remotely."
 	icon_state = "holopad0"
+	layer = 2.1

```

Comment:
yes, it is.  It makes sense, too me at least, that drawings would go over it, seeing as it is set in the ground.